crane-cloud/backend
# Add soft delete to Apps and Databases Models

# Description
Adds Soft delete to App and Database Models.
On soft delete the app and database names are concatenated with `+_deleted_{timestamp}` and a field `deleted` is set to true.

## How Can This Be Tested?

When you delete an app or project, you shouldn't see it as per the previous flow. But with the current implementation, we expect the same results but rather when you check the database the record of the deleted **App** or **Database** should exist.


## Checklist:
